tests/revalidate-dashboard.service.test.ts

```
@@ -0,0 +1,145 @@
+import { buildSlug } from "@/utils/slug-hash";
+
+interface CapturedRequest {
+  url: string;
+  method: string;
+  headers: Record<string, string>;
+  body: unknown;
+}
+
+let captured: CapturedRequest[] = [];
+let fetchImpl: (url: string, init: RequestInit) => Promise<Response> = async () =>
+  new Response(null, { status: 200 });
```

ESLint

[error] Replace `url:·string,·init:·RequestInit)·=>·Promise<Response>·=·async·()·=>⏎·` with `⏎··url:·string,⏎··init:·RequestInit,⏎)·=>·Promise<Response>·=·async·()·=>`

(prettier/prettier)